### ISSUE TITLE
What happend to Eric Douglas?

### ISSUE THREAD

**Issue by MirzaI** [open-source-society/computer-science#253]
The guy who created the project, [Eric Douglas](https://github.com/ericdouglas) did not have any activity since late November 2015.

Is he done with this project? It's a pity that such a great project is getting abandoned.

**Comment by QuincyLarson** [comment]
I haven't talked with him recently on Gitter. I agree - this project has a ton of potential. @ericdouglas if you're around, please let us know.

**Comment by NajumRahim** [comment]
Not even seen his activity on twitter.

**Comment by NajumRahim** [comment]
Not even seen his ( @ericdouglas )activity on Twitter

**Comment by rands0n** [comment]
Guys, perhaps something happens to him. It does not answer my emails since november.

**Comment by bestknighter** [comment]
IDK but if he is in the last semester of his major, he may have no time to do anything. This is because in this moment (at least in Brazilian universities) we have to do a HUGE paper that swallows all our time (be it free time or not).
I really hope that's the case. I'm worried too.

**Comment by Ebisu77** [comment]
He is also silent on reddit from late november... 

https://www.reddit.com/user/eric-douglas/

**Comment by mseyne** [comment]
I am worried too, please give news if someone have hear from him.

**Comment by olgamikks** [comment]
Please keep informed of any news.

**Comment by sand33pn** [comment]
He was too active on his projects to suddenly abandon them without letting anyone know. 
There must be some way to make contact.

**Comment by rmdias** [comment]
On him linkedin @ericdouglas  is working at @pagarme I'll try find someone there to know about this guy.

Cc @chrisenytc , @hdubugras and @pedrofranceschi
https://www.linkedin.com/in/eric-douglas-81381282/en

**Comment by chrisenytc** [comment]
I don't hear about him. I think he is following the example of John McAfee. Moved to Guatemala and created a methamphetamine lab.

**Comment by bestknighter** [comment]
I contacted some of the other OSS organization members asking about him and they don't know, also. But they said that he may be busy with a big project and then will come back later.

**Comment by MirzaI** [comment]
@rmdias Thank you for taking the initiative to contact his workplace.

**Comment by mseyne** [comment]
I was member of the organization, he shut down the project and left few time after. I hope it is not related and he is fine.

**Comment by jdinartejesus** [comment]
Any news about @ericdouglas? Or still missing?

**Comment by rands0n** [comment]
Still missing! No answers here and email. I don't have his phone to call.  
  

\---

Randson Oliveira

Sent from Nylas Mail Client

> On Feb 3 2016, at 6:43 am, Dinarte Jesus &lt;notifications@github.com&gt;
wrote:  

>

> Any news about [@ericdouglas](https://github.com/ericdouglas)? Or still
missing?

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/open-
source-society/computer-science/issues/253#issuecomment-179094047).![](https:/
/github.com/notifications/beacon/AD_19v8orPqdrfQbVSYWwHeSHTbxMRCtks5pgbUZgaJpZ
M4HHuEA.gif)

**Comment by mseyne** [comment]
Emails don't answer and the disapearance is too sudden to look like being busy on any others projects, it just vanishing from internet presence. Though I hope it is just that. Eric have a good heart and I hope everything is fine for him.

**Comment by QuincyLarson** [comment]
@ericdouglas if you are reading this, please respond to this thread. We're worried about you, man! Hope everything is well with you and your family!

**Comment by ni8mr** [comment]
Following this thread. I hope he is **ok **and this project will continue towards greater potentiality.

**Comment by crmackay** [comment]
I would say this is going beyond slightly concerning now ...

![image](https://cloud.githubusercontent.com/assets/3801862/13159637/e859a1e4-d660-11e5-9b5e-475aa49c11b9.png)

... is there anything we can do? reach out to his family, somehow?

I've tried using google to find any clues but came up empty handed. The timestamp on his last github commit is 

```
commit ca61a9805f36d1a673634a154b26e76e2a7e7b49
Author: Eric Douglas <eric.douglas.mail@gmail.com>
Date:   Thu Nov 26 17:34:41 2015 -0200
```

**Comment by jdinartejesus** [comment]
@crmackay I found his facebook but also don't have activity since 31 May 2015.

**Comment by mseyne** [comment]
Maybe we should move this thread somewhere else for a better visibility, maybe reddit ? brasilian javascript community ? If someone have an idea and speak portuguese ?

**Comment by Davkilla101** [comment]
Why not both? We could try posting this on /r/programming (might have to talk to the mods first) and you don't really need to speak Portuguese for Brazilian people to understand what you're saying, as Spanish is close enough to Portuguese that you can hold a conversation with them.

**Comment by jdinartejesus** [comment]
@Davkilla101 @mseyne  Well I can help, i speak Portuguese. Today i'll try to send a message to one for his friends on Facebook from Brazil.

**Comment by Davkilla101** [comment]
@jdinartejesus please do

**Comment by mseyne** [comment]
@jdinartejesus :+1: 

I did a reddit post [here](https://www.reddit.com/r/javascript/comments/46kw20/js_community_please_help_us_to_solve_this_mystery/) but not sure the moderator accept it.

But you right, facebook contact with his friends is probably the best solution to have news. I don't have facebook and don't speak portuguese, than maybe, some friend around him did say something about it on facebook already.

**Comment by jdinartejesus** [comment]
@mseyne @Davkilla101 Already send the private message to one of colleague or ex-colleague of work. Let's hope for the best. I'll let you know when i have some news.

**Comment by crmackay** [comment]
sounds good.

I know we could just let it go, and move on, but I personally feel like we owe Eric some form of recognition for all the work he has done (on this project and others) and the tremendous impact he has had on so many (just look at the 1000+ people who signed up for OSSU-CS so far). 

There are a lot of potential explanations for his absence, but obviously our minds jump to the worst ones...maybe once we have an answer we can find a way to show our support for him and/or his family.

**Comment by idchlife** [comment]
I came from reddit, can't really help with anything (I'm far away from Eric), but I agree with @crmackay,
we need to investigate, especially when we might be those people who will find him and maybe this will be his chance, if he is in some kind of situation there.

**Comment by rfviolato** [comment]
I had the pleasure to work with him and start a project with him, but unfortunately since the end of november from last year he just disappeared :/

I really hope he's alright, he is a dear friend and one of the most dedicated developers i have ever known :(

**Comment by dlphi** [comment]
Maybe is better to try to find his skype and call there or contact his Uni department

**Comment by rfviolato** [comment]
I contacted the Human Resources at Pagar.me, the company i have worked with him, i'll try to get his house number.

I'll try to update you guys with any info i can get.

**Comment by rands0n** [comment]
Thanks @rafaelviolato
—
Randson Oliveira

**Comment by Davkilla101** [comment]
Thank you, @rfviolato please let us know what they say

**Comment by idchlife** [comment]
Thank you @rfviolato!

**Comment by bestknighter** [comment]
Wow, we have activated the internet's power of investigation. I think, and hope so, we may have an answer to this mystery soon.

**Comment by irazasyed** [comment]
HN bought me here.  

Hope things are alright at @ericdouglas end and we'll have some news soon.

Thanks @rfviolato.

**Comment by paragonie-scott** [comment]
Has it occurred to anyone in this thread that it might be possible that Eric Douglas doesn't want to be found?

**Comment by Davkilla101** [comment]
It's possible @patagonie-scott , though what reason would he have to not want to be found? I mean, I don't think he's being hunted by the mob or anything.

**Comment by rfviolato** [comment]
Ok guys i got his cellphone, that's all information i could get from him from the HR at pagarme.

I tried to call him and it always rang twice than the person who had the cellphone refused the call. On my last try i could get the call to ring many time before going to the voice mail.

I sent an SMS, also left the number a voice message, no response yet.

I have also tried to contact his girlfriend via Facebook, no return yet.

I feel like im getting closer. Let's wait for the returns.

**Comment by rfviolato** [comment]
@paragonie-scott That's always a possibility but personally, as a close friend, i doubt it.

**Comment by Davkilla101** [comment]
@rfviolato, that's kind of odd that the person with the phone refused the calls like that

**Comment by rfviolato** [comment]
Yeah, his register at pagarme must be like an year old, there is a possibility that this is not his number anymore.

Anyway, that's the closest info i could get from the HR.

**Comment by Davkilla101** [comment]
Have you tried a reverse phone lookup to see who it's registered to?

**Comment by mseyne** [comment]
I understand people who think about  all this being a hunt, but for me it is just about knowing he is fine and I will respect his privacy if he decided to take a break from internet. I can understand that.

**Comment by idchlife** [comment]
Guys I will try to read his responses on reddit. Maybe there will be some kind of mention where he was lastly. I think at some point we might want to contact local people somehow (reddit, police etc) to find him.

About this, @paragonie-scott - as soon as we contact him and know he does not want to be found - we will stop.

I'm afraid that he is in some kind of trouble and we all can help somehow.

**Comment by idchlife** [comment]
@rfviolato do you have information on where he is living? I mean country/city smth?

**Comment by Davkilla101** [comment]
@idchlife IIRC he is in Brazil

**Comment by rfviolato** [comment]
Guys, his girlfriend just commented on a thread on his facebook timeline, she said he is alive.

![image](https://cloud.githubusercontent.com/assets/5693916/13198610/bf4f5a56-d7f4-11e5-8c92-41107ad0b1eb.png)

In english would be something like: "Lol, he's alive guys!!!".

Well i guess he's just taking a long break guys, let him take his time for whatever he decides to do, i just hope he comes back coding because he used to be one of the best programmers i have ever known and worked with :/

**Comment by rougin** [comment]
Thank you @rfviolato , that's nice to hear. :+1:

**Comment by idchlife** [comment]
@rfviolato please keep us updated on any new info before he will come back to us :) We don't have access and we (I, personally) do not know spanish :|

Thank you!

**Comment by jsvgoncalves** [comment]
@idchlife sucks that you don't know Spanish and therefore can't understand brasilian people, 1 feel for you

**Comment by rands0n** [comment]
heheheh we speak english. but not all of us :(
—
Randson Oliveira

**Comment by esnard** [comment]
@idchlife in case you don't understand the irony behind @jsvgoncalves's post, Brazilian people speak Portuguese. :P

**Comment by MirzaI** [comment]
Don't see any reason for this thread to continue further.

tldr:
Eric is alive according to his girlfriend.

**Comment by bestknighter** [comment]
We just witnessed the power of the internet and the theory of 7 degrees of distance to anyone happening right in front of our eyes. This is awesome! Thankfully he's ok and ~~I hope~~ we hope he get back soon. That's a relief...

**Comment by ericdouglas** [comment]
@MirzaI I'm here again! :tada: 

The project won't be abandoned guys!

I truly appreciate all of your preoccupation about me. I'm well and as I say, I'm here again to help all of the OSSU students as much as I can.

Let's improve this project together and help lots of students from all the world!

**Comment by bestknighter** [comment]
HE'S BACK! Everyone, look at this! Welcome back, @ericdouglas! :tada: :tada: :tada:

**Comment by jdinartejesus** [comment]
Welcome back @ericdouglas

**Comment by ericdouglas** [comment]
@bestknighter @jdinartejesus  thank you so much guys :smile:

**Comment by MirzaI** [comment]
The king is back!

**Comment by ni8mr** [comment]
Welcome back !

**Comment by wakenmeng** [comment]
Welcome back!

2016-03-06 17:05 GMT+08:00 Noor Faizur Reza <notifications@github.com>:

**Comment by idchlife** [comment]
@ericdouglas welcome back! We glad to see you with us!

**Comment by 007lva** [comment]
Welcome back!

**Comment by rougin** [comment]
Welcome back @ericdouglas !

**Comment by Davkilla101** [comment]
Welcome back @ericdouglas, good to see you alive and well :D

**Comment by NajumRahim** [comment]
Welcome back man @ericdouglas Glad to see you!!

**Comment by ericdouglas** [comment]
Thank you guys! :pray:

**Comment by franklinjavier** [comment]
## Hey everyone... I have good news 🎉
He is alive and working here at @belezanaweb

![image](https://user-images.githubusercontent.com/440220/62165206-4de30a00-b2f4-11e9-9e1d-7efa1f0f42f4.png)

**Comment by diogomoretti** [comment]
Essa comunidade é demais ❤️ BTW: Saudades @ericdouglas
